src/server.ts

```diff
@@ -4,9 +4,10 @@ import { getContext } from './context'
 
 export async function listen () {
   const ctx = getContext()
-  const port = await getPort()
+  const { server } = ctx.options.config
+  const port = server?.port ? Number(server.port) : await getPort()
```

Comment:
We can pass server.port as [default port](https://www.npmjs.com/package/get-port) otherwise an error happens if port is already used

```suggestion
  const port = await getPort({ port: server?.port?.port })
```